docs/mint.json

```diff
@@ -82,47 +82,53 @@
         {
```

Comment:
why all of these changes?

Reply:
Removed formatting, addded the providers I've added till now, and sorted them